.github/workflows/main.yml

```diff
@@ -195,7 +213,7 @@ jobs:
         name: release-notes.md
         path: release-notes.md
     - name: GitHub Release
-      uses: ncipollo/release-action@v1
+      uses: ncipollo/release-action@440c8c1cb0ed28b9f43e4d1d670870f059653174  # v1.16.0
```

Comment:
Not sure if it's wise to give this action OIDC privileges..

Reply:
I understand your concern here. Do you see an alternative approach for creating GitHub releases as part of a package release? If we have similar without the action, we should consider removing it.

Other options:
- Rethink the release process entirely
- Review the pinned version of the action